.eslintrc.json

```diff
@@ -39,6 +39,8 @@
 
         "no-console": "off",
         "quotes": "off",
+	"key-spacing": "off",
+	"object-property-newline": "off",
```

Comment:
Hmm, IMHO it would be better to entirely ignore pkg/lib/ (just like node_modules) in eslint for all cockpit extensions. But that's not a blocker.

Reply:
Oh, good point! Meanwhile this is underway https://github.com/cockpit-project/cockpit/pull/17876 Let's make it consistent everywhere!

Reply:
Well, that was easier then expected and we do this already in cockpit-podman...